test/CMakeLists.txt

```diff
@@ -2,16 +2,14 @@ add_subdirectory(src)
 
 enable_testing()
 add_definitions(-DREPO_API_LIBRARY)
-
+# override as we do not want to impose license checks on the test project
+remove_definitions(-DREPO_LICENSE_CHECK)
 
 include_directories(${gtest_SOURCE_DIR}/include ${gtest_SOURCE_DIR} src ../bouncer/src ../client/src ../
-	${Boost_INCLUDE_DIR} ${MONGO_INCLUDE_DIR} ${ASSIMP_INCLUDE_DIR}  ${OCCT_INCLUDE_DIR} ${IFCOPENSHELL_INCLUDE_DIR}  ${ODA_INCLUDE_DIR} ${AWSSDK_INCLUDE_DIR}  ${SYNCHRO_READER_INCLUDE_DIR} )
+	${Boost_INCLUDE_DIR} ${MONGO_INCLUDE_DIR} ${ASSIMP_INCLUDE_DIR}  ${OCCT_INCLUDE_DIR} ${IFCOPENSHELL_INCLUDE_DIR}  ${ODA_INCLUDE_DIR} ${AWSSDK_INCLUDE_DIR}  ${SYNCHRO_READER_INCLUDE_DIR})
 add_executable(3drepobouncerTest ${TEST_SOURCES} ${SOURCES})
 target_link_libraries(3drepobouncerTest gtest
-	${Boost_LIBRARIES} ${MONGO_LIBRARIES} ${ASSIMP_LIBRARIES} ${OCCT_LIBRARIES} ${IFCOPENSHELL_GEOMLIB} ${IFCOPENSHELL_PARSERLIB}  ${ODA_LIB}  ${SYNCHRO_READER_LIBRARIES} ${THRIFT_LIBRARIES} ${ZLIB_LIBRARIES} ${SYNCHRO_LIBRARIES} ${AWSSDK_LIBRARIES} ifcUtils_2x3 ifcUtils_4)
+	${Boost_LIBRARIES} ${MONGO_LIBRARIES} ${ASSIMP_LIBRARIES} ${OCCT_LIBRARIES} ${IFCOPENSHELL_GEOMLIB} ${IFCOPENSHELL_PARSERLIB}  ${ODA_LIB}  ${SYNCHRO_READER_LIBRARIES} ${THRIFT_LIBRARIES} ${ZLIB_LIBRARIES} ${SYNCHRO_LIBRARIES} ${AWSSDK_LIBRARIES} ifcUtils_2x3 ifcUtils_4 Winhttp.lib)
```

Comment:
This isn't adding in the license libs, we should just add it as part of the license lib (if windows)

Reply:
you'd also need to check bouncerTools and bouncerWrapper are compiling correctly.